aws/resource_aws_kms_external_key.go

```diff
@@ -249,7 +250,12 @@ func resourceAwsKmsExternalKeyRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("key_usage", metadata.KeyUsage)
 	d.Set("policy", policy)
 
-	if err := d.Set("tags", tagsToMapKMS(listResourceTagsOutput.Tags)); err != nil {
+	tags, err := keyvaluetags.KmsListTags(conn, d.Id())
```

Comment:
Since this is performing the same `kms.ListResourceTags` call as line 235 above, will remove the extraneous handling above on merge.